dl1_data_handler/reader.py

```diff
@@ -198,8 +198,7 @@ def __init__(self,
                     self.pixel_positions[cam] = np.squeeze(np.asarray(np.dot(rotation_matrix, self.pixel_positions[cam])))
         if rotate_back:
             mapping_settings['rotate_back'] = rotate_back_angle[cam]
-        self.image_mapper = ImageMapper(pixel_positions=self.pixel_positions,
-                                        **mapping_settings)
+        self.image_mapper = ImageMapper(pixel_positions=self.pixel_positions, camera_types = cameras, **mapping_settings)
```

Comment:
Can you add the returns here? It would be easier to read.

Reply:
And you need to init 'cameras' with None before line 183. Otherwise it will break if  `/Telescope_Type_Information` is not in h5.

Reply:
Sure!